src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -577,6 +577,12 @@ public string Style_UseImplicitTypeWherePossible
             set { SetXmlOption(CSharpCodeStyleOptions.UseImplicitTypeWherePossible, value); }
         }
 
+        public string Style_UseBracesWherePossible
```

Comment:
nit: consider grouping various options of the same style together. for e.g: this is now in between `use implicit` options. not a concern, but just a preference, if possible.